geometry_msgs/PoseArray[] vertices_list # vertices list of the detected object
